modules/drivers/athena/resources/metabase-plugin.yaml

```
@@ -16,7 +16,7 @@ driver:
       display-name: Workgroup
       default: "primary"
     - name: s3_staging_dir
-      display-name: S3 Staging dir
+      display-name: S3 staging directory
```

```suggestion
      display-name: S3 staging directory
      helper-text: This S3 staging directory must be in the same region you specify above.
```
From a note in the original repo: https://github.com/dacort/metabase-athena-driver